### ISSUE TITLE
Copy of a single wrapped line from bash introduces new-lines

### ISSUE THREAD

**Issue by vadimkantorov** [microsoft/terminal#14035]
### Windows Terminal version

 1.14.2282.0

### Windows build number

10.0.22000.978

### Steps to reproduce

I have launched a long command in WSLv1 bash in Terminal, it looks like:
```bash
vadimkantorov@dell_de_vadim:/mnt/c/Users/vadim/$ python ..... my long arguments
long arguments wrapped by terminal window size but still a single line
```

When I copy these two lines (e.g. for pasting later in another terminal for launching a similar command in parallel), Terminal introduces a newline which was not there originally. This prevents from a proper paste and (probably another issue) - also executes the incomplete command immediately after the paste. It's possible that I entered this command by pressing Up on keyboard which brought this command from bash history.

This must have been reported before, but I could not find an earlier issue just about this. If it indeed is a duplicate, please feel free to close this.

### Actual Behavior

No newline is introduced and command is not executed immediately after the paste in another tab

**Comment by DHowett** [comment]
This is a bit nuanced.

bash (more accurately, probably readline) implements its own hard wrapping. As a result, we can't totally tell that a line was intended to be a continuation instead of a physical line break.

When I type this:

<img width="850" alt="image" src="https://user-images.githubusercontent.com/189190/191073975-c0b3d492-dc54-4644-ac94-719358218a13.png">

Bash sends this:

```
reach the edge ABCDEFGHIJKL:M \nNOP
                             ^^^
```

That space and newline were produced by them. I'm at a loss as to exactly _why_ they do this. ☹️

**Comment by vadimkantorov** [comment]
probably the command is actually executed because of this newline too

**Comment by vadimkantorov** [comment]
maybe could be hacked around by a context menu command "Paste without newlines", but it's not very satisfactory

**Comment by DHowett** [comment]
Holding down shift while you copy will force the entire selection to wrap onto a single line. It's not ideal because you won't necessarily know when to use it versus not use it, but it might be a good tactical solution for this specific issue. Sorry :/

**Comment by vadimkantorov** [comment]
Until now, I've been relying on auto-copy on mouse selection. I wonder, if Terminal can at least provide some visual hints on copy or on paste if the text includes new-lines to educate about this option. Btw Ctrl+Shift+C is default on Gnome Terminal, will this shortcut in WT also just apply these newline removal? or only if Shift is used during mouse-selection?

**Comment by vadimkantorov** [comment]
> That space and newline were produced by them. I'm at a loss as to exactly _why_ they do this. ☹️

Yea, I wonder if there's somewhere a mailing list or bash/readline authors that could address this...

**Comment by zadjii-msft** [comment]
without properly triaging:
* #6901
* #6987
* #8976
* #9359
* #9933
* #11459

**Comment by zadjii-msft** [comment]
I think I'm gonna call this a dupe of #6901. We should probably go through and de-dupe the rest of those too.
